style.css

```diff
@@ -0,0 +1,485 @@
+* {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  color: #344563;
+}
+
+body {
+  width: fit-content;
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-x: hidden;
+}
+
+header {
+  background-color: white;
+  width: 100%;
+}
+
+.backgroundimg {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/image/background.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: white;
+  border-radius: 0 0 0 120px;
+  height: 680px;
+  width: 100%;
+}
+
+#cards {
+  display: grid;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
- [ ] Good job using `display: grid` to align your project cards 👍🏼 as I have suggested earlier . I think you should add `grid-template-columns: 1fr 1fr;`  and  `align-content: center`  to align the cards properly on the page. You should also add padding to the container as the cards are touching the edges of the viewport 😄 

-Kindly ensure that your project card layout matches the Figma design template chosen  👍🏼 . Kindly add the appropriate padding, and position the elements properly just as I have suggested earlier👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/185760602-51e02053-2a11-4702-83a7-a0f782d009d8.png)  | ![image](https://user-images.githubusercontent.com/63473584/185765001-468913f3-3d32-4628-ad68-acf52a5fe28a.png)  |

Reply:
As you may know, I am running out of time and I see you've added some reviews that weren't included in the previous review which isn't the normal procedure